### DISCUSSION TITLE
Facing issue while setting up the new workspace

### DISCUSSION THREAD

**Discussion by Dinesh0723** [eclipse-platform/eclipse.platform.swt#1025] [Q&A]
I am encountering the error 'java.lang.Exception: org.eclipse.jgit.api.errors.TransportException: Premature end of chunk coded message body: closing chunk expected' while setting up the new workspace. Has anyone else experienced this issue?

**Comment by merks** [comment]
Is this a different question from https://github.com/eclipse-platform/eclipse.platform/discussions/1170?  How can I reproduce this issue?

**Comment by Dinesh0723** [comment]
Yes, it's a different issue encountered while setting up the SWT workspace.

Steps to Reproduce:

1. Download and install the Oomph installer .exe, then switch to advanced mode.
2. Select the product (Eclipse IDE for Committers) and choose the latest version from the product version combo box.
3. Click on the Network Proxy Settings icon and add the necessary proxy details.
4. Choose the Eclipse projects from the checkbox tree viewer (eg.: SWT).
5. Select the installation folder, then click "Next" and subsequently "Finish".

During installation, the process is interrupted, and I encounter the error 'java.lang.Exception: org.eclipse.jgit.api.errors.TransportException: Premature end of chunk coded message body: closing chunk expected'.

**Comment by merks** [comment]
That sounds like a network problem.  It could also be related to the large file system support not configured properly, but it does get installed:

![image](https://github.com/eclipse-platform/eclipse.platform.swt/assets/208716/c102fdc0-bfc8-42b7-9bb1-1952504dff48)

It does work for me:

![image](https://github.com/eclipse-platform/eclipse.platform.swt/assets/208716/b77810e1-736f-46c2-b520-f5b2a87738fa)

**Reply by merks** [reply]
Is it only SWT that's failing?  Then it's likely related to the large file support....

**Comment by Dinesh0723** [comment]
As mentioned, I followed steps to set up the SWT workspace. The setup is complete (without any compilation errors), and now I am able to launch the runtime eclipse application. However, when I check out the latest SWT source in the SDK, the problem (https://github.com/eclipse-platform/eclipse.platform.ui/issues/1466) does not occur. Nonetheless, upon closing the checked-out SWT source projects from Eclipse SDK (Version: 2024-06 (4.32), Build id: I20240316-1800), the issue reappears. Has anyone faced this kind of issues in SWT?

> Yes, it's a different issue encountered while setting up the SWT workspace.
> 
> Steps to Reproduce:
> 
> 1. Download and install the Oomph installer .exe, then switch to advanced mode.
> 2. Select the product (Eclipse IDE for Committers) and choose the latest version from the product version combo box.
> 3. Click on the Network Proxy Settings icon and add the necessary proxy details.
> 4. Choose the Eclipse projects from the checkbox tree viewer (eg.: SWT).
> 5. Select the installation folder, then click "Next" and subsequently "Finish".
> 
> During installation, the process is interrupted, and I encounter the error 'java.lang.Exception: org.eclipse.jgit.api.errors.TransportException: Premature end of chunk coded message body: closing chunk expected'.
